server/integration_tests/test_data/detection_api_context/stateswithhighestphds/chart_config.json

```diff
@@ -1,34 +1,16 @@
 {
-  "childEntityType": "State",
```

Comment:
I'm not sure what change would've caused this, but tried running this on prod and got the same result: https://paste.googleplex.com/5743034006765568

Reply:
Looks like this hits LLM and it is blocking.  Might be OK for now, and I will try to switch to using Gemini API, instead of PaLM.

Reply:
ok sounds good, I will merge this